src/test/java/com/google/sps/PostTest.java

```diff
@@ -0,0 +1,195 @@
+package com.google.sps;
+
+import com.google.sps.data.Post;
+import javax.servlet.http.HttpServletRequest;
+import org.junit.Assert;
+import org.junit.runners.JUnit4;
+import org.junit.runner.RunWith;
+import org.junit.Test;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public final class PostTest {
+
+  // Given inputs for an HTTP Request, tests if the conversion to a Post object is valid.
+  // Tester must check if start/end hour and month are valid manually.
+  public Post testRequestToPost(String collegeId, String organizationName, int month, int day, int startHour, int startMinute, String startAMorPM, int endHour, 
+  int endMinute, String endAMorPM, String location, Double lat, Double lng, String numberOfPeopleItFeeds, String typeOfFood, String description) {
```

Comment:
A builder pattern would be much better here. 

Maybe you can add a TODO for now to fix this. Since with so many variables, it can be hard to know which goes where.